docs/cloud/pouta/persistent-volumes.md

```diff
@@ -64,6 +64,10 @@ Then you can mount it:
 
     sudo mount /dev/vdb /media/volume
 
+you can also check the status of the enlarged volume of the instance's new size  in a mounted XFS filesystem using the following command:
```

Comment:
```suggestion
You can also check the status of the enlarged volume of the instance's new size in a mounted XFS filesystem using the following command:
```
Small change in formatting.
"status of the enlarged volume of the instance's new size" could maybe be reworded as it is somewhat confusing.
Could be something simple as "status of the volume in a mounted XFS filesystem".